systemd/mullvadvpn-app-tests.service

```diff
@@ -0,0 +1,11 @@
+# Systemd unit for running app tests and sending a status report
+# to the given set of email addresses.
+#
+[Unit]
+Description="Mullvad VPN app tests"
+
+[Service]
+Type=oneshot
+ExecStart=/home/test/mullvadvpn-app-tests/ci-runtests-and-report.sh
+Environment="ACCOUNT_TOKEN=XXXXXXXXXXXXXX"
```

Comment:
Bit unclear how do we get the actual number in there? Do we change this file without commiting it to git?

Reply:
Yes. There are instructions in `README.md`.